implementation-jaxrs2/src/main/java/io/smallrye/opentelemetry/jaxrs2/client/TracingProperties.java

```diff
@@ -0,0 +1,25 @@
+package io.smallrye.opentelemetry.jaxrs2.client;
+
+/**
+ * @author Pavol Loffay, Felix Wong
+ */
+public class TracingProperties {
```

Comment:
Either document the features in readme and add tests or remove this. 

https://github.com/opentracing-contrib/java-jaxrs#tracing-client-requests